var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -285,6 +285,23 @@ class NMakeBuilder(spack.build_systems.nmake.NMakeBuilder):
     def makefile_name(self):
         return "Makefile.msc"
 
+    def nmake_args(self):
+        return [
+            "USE_NATIVE_LIBPATHS=1",
```

Comment:
I'm assuming this is the

> Sqlite requires the user to provide a command line arg to export shared symbols to import lib from .def

part

and the rest is

> Use command line nmake args used by official sqlite shared distribution + export arg (sqlite only distributes dll + .def)

which I assume comes from 24.1 of https://sqlite.org/cli.html

but it seems like this is doing at least a couple extra things which I don't see explained in your PR desciption, like `OPTS=`.

If you have a link that explains all this, can you add it here (so I could include it in the merge comment)?

Reply:
`DYNAMIC_SHELL` is 
> Sqlite requires the user to provide a command line arg to export shared symbols to import lib from .def

Everything else is
> Use command line nmake args used by official sqlite shared distribution

from the Sqlite docs at https://github.com/sqlite/sqlite/blob/master/doc/compile-for-windows.md

Reply:
Thanks, I think this will make good info for the commit message